src/main/java/qna/domain/BaseEntity.java

```diff
@@ -0,0 +1,31 @@
+package qna.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+@EntityListeners(value = {AuditingEntityListener.class})
+public abstract class BaseEntity {
```

Comment:
audit 활용을 위한 `BaseEntity` 좋습니다! 😁

```suggestion
public class BaseEntity {
```

그리고 위 부분을 살펴보면 저는 어노테이션으로 이미 이 클래스의 역할을 충분히 알렸다고 생각해요.
abstract method 없는 abstract 는 불필요하다고 생각하는데 상재님은 어떻게 생각하시나요?
그리고 리뷰 요청하시는 분들이 abstract 를 많이 활용하시던데 혹시 JPA 강의에 그런 지침이 있었을까요? 🤔

Reply:
`@MappedSuperclass` 가 선언된 클래스는 단순히 자식클래스에 매핑 정보를 상속할 목적으로만 사용하고,
직접 생성해서 사용할 일이 없기 때문에 추상 클래스로 만드는것을 권장한다고 알고 있습니다!

이번에 JPA를 공부하면서 김영한님의 `자바 ORM 표준 JPA 프로그래밍` 책을 보고있는데,
이 책의 예제에서도 `abstract` 클래스로 사용하고 있어서 다른 분들도 그렇지 않을까 생각합니다. 🤔